judge/models/problem.py

```diff
@@ -23,6 +24,12 @@
            'License', 'Solution', 'TranslatedProblemQuerySet', 'TranslatedProblemForeignKeyQuerySet']
 
 
+def disallowed_characters_validator(text):
+    for c in text:
```

Comment:
`if set(text) & set(settings.DMOJ_PROBLEM_STATEMENT_DISALLOWED_CHARACTERS)` is probably faster, especially when the setting is turned into a set.

```console
$ python3 -m timeit -s "d = ('“', '”', '‘', '’'); t = '<snippet from Wikipedia>'" $'for c in t:\n if c in d:\n  break'
10000 loops, best of 5: 30.9 usec per loop
$ python3 -m timeit -s "d = ('“', '”', '‘', '’'); t = '<snippet from Wikipedia>'" 'set(t) & set(d)'
50000 loops, best of 5: 5.8 usec per loop
$ python3 -m timeit -s "d = {'“', '”', '‘', '’'}; t = '<snippet from Wikipedia>'" 'set(t) & d'
50000 loops, best of 5: 5.27 usec per loop